lib/coverband/adapters/base.rb

```diff
@@ -57,8 +60,9 @@ def expand_report(report)
             'file_hash' => file_hash(key),
             'data' => line_data
           }
-          report[key] = extended_data
+          expanded[full_path_to_relative(key)] = extended_data
```

Comment:
When I first looked at the concepts here I got a little confused. We are using the word expanded to expand the data we are storing but converting the files from full path to relative path. Possible sign that this method is doing more than 1 thing or that my brain can't handle very much 😏

Reply:
yeah, this in our tight reporting loop I am trying to stay with as few iterations as possible through the collection. As this method has large impact on the perf. I think renaming the method could help clarify but hard to avoid doing more than one thing